src/main/osd/osd_elements.c

```diff
@@ -1060,10 +1060,14 @@ static void osdElementFlymode(osdElementParms_t *element)
         strcpy(element->buff, "HEAD");
     } else if (FLIGHT_MODE(PASSTHRU_MODE)) {
         strcpy(element->buff, "PASS");
+    } else if (FLIGHT_MODE(POS_HOLD_MODE)) {
+        strcpy(element->buff, "POSH");
     } else if (FLIGHT_MODE(ANGLE_MODE)) {
         strcpy(element->buff, "ANGL");
```

Comment:
```suggestion
```

Reply:
Thanks, don't know how that appeared again.  Agree that Alt Hold and Pos Hold should precede Angle.